runtime/compiler/runtime/RelocationRecord.hpp

```diff
@@ -1445,6 +1445,17 @@ class TR_RelocationRecordValidateProfiledClass : public TR_RelocationRecord
       virtual int32_t bytesInHeaderAndPayload() { return sizeof(TR_RelocationRecordValidateProfiledClassBinaryTemplate); }
       virtual void preparePrivateData(TR_RelocationRuntime *reloRuntime, TR_RelocationTarget *reloTarget) {}
       virtual int32_t applyRelocation(TR_RelocationRuntime *reloRuntime, TR_RelocationTarget *reloTarget, uint8_t *reloLocation);
+
+      virtual void print(TR_RelocationRuntime *reloRuntime);
+
+      void setClassID(TR_RelocationTarget *reloTarget, uint16_t classID);
+      uint16_t classID(TR_RelocationTarget *reloTarget);
+
+      void setClassChainOffset(TR_RelocationTarget *reloTarget, uintptr_t classChainOffset);
+      uintptr_t classChainOffset(TR_RelocationTarget *reloTarget);
+
+      void setClassChainOffsetForCL(TR_RelocationTarget *reloTarget, uintptr_t classChainOffsetForCL);
```

Comment:
Given the number of places these two are called, I would probably have just spelled out `ClassLoader` so others don't have to think about what "CL" should stand for.

Reply:
I added the full form of CL in https://github.com/eclipse/openj9/pull/10298/commits/4fef70154a406826c772815ecf8e1433c45a4239 since it's probably better to be explicit.